releasetasks/templates/firefox/l10n.yml.tmpl

```diff
@@ -355,6 +357,12 @@
 {% if push_to_candidates_enabled %}
 {% set partial_beetmover_buildername = "{}_partial_{}build{}_beetmover_candidates_{}".format(buildername, partial_version, partial_info["buildNumber"], chunk) %}
 {% do artifact_partials_builders.append(partial_beetmover_buildername) %}
+{% set chunk_locales = [] %}
+{% for l in our_locales %}
+{% if l in partial_info["locales"] %}
+{% do chunk_locales.append(l) %}
```

Comment:
wonder if this var should be at the top and we reuse that instead of `if locale in partial_info["locales"]` at line 170?